src/test/java/core/basesyntax/ReportServiceImplTest.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax;
+
+import core.basesyntax.database.Storage;
+import core.basesyntax.service.ReportService;
+import core.basesyntax.service.impl.ReportServiceImpl;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ReportServiceImplTest {
+    private static ReportService reportService;
+    private static Storage storage;
+    private static StringBuilder validReport;
+    private static StringBuilder validReportEmptyFile;
+
+    @BeforeClass
+    public static void setUp() {
+        reportService = new ReportServiceImpl();
+        storage = new Storage();
+        storage.storage.put("banana", 25);
+        storage.storage.put("apple", 25);
+        validReport = new StringBuilder();
+        validReport.append("fruit,quantity");
+        validReport.append(System.lineSeparator());
+        validReport.append("banana,25");
+        validReport.append(System.lineSeparator());
+        validReport.append("apple,25");
+        validReport.append(System.lineSeparator());
+        validReportEmptyFile = new StringBuilder();
+        validReportEmptyFile.append("fruit,quantity");
+        validReportEmptyFile.append(System.lineSeparator());
+    }
+
+    @After
+    public void tearDown() {
+        storage.storage.clear();
+    }
+
+    @Test
+    public void generateReportEmptyStorage_Ok() {
+        String expected = validReportEmptyFile.toString();
+        String actual = reportService.generateReport();
+        Assert.assertEquals(expected,actual);
+    }
```

Comment:
Incorrect name.  What will happen if I remove this test case? Your test cases have to independent

Reply:
I removed this test for watch what happen and it seems to be ok. Please explain why this test is not independent and what I need to fix.